src/main/java/hng_java_boilerplate/user/controller/UserController.java

```diff
@@ -30,12 +30,24 @@ public ResponseEntity<?> getUserDetails(@PathVariable String userId) {
     @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
     @GetMapping(value = "/members", produces = "application/json")
     public ResponseEntity<?> getAllMembers(@RequestParam int page, Authentication authentication) {
+        if (authentication == null) return ResponseEntity.status(HttpStatus.UNAUTHORIZED)
```

Comment:
As long as the route is protected, the authentication object will never be null which also means it's not needed